spec/features/merchants/dashboard_spec.rb

```diff
@@ -113,14 +113,16 @@
       end
     end
   end
-end
 
-# 5. Merchant Dashboard Invoices sorted by least recent
+  # User stories 37 and 40
+  it "has the logo and likes" do
+    visit merchant_path(@merchant1)
+
+    image_src = "https://images.unsplash.com/photo-1666324574241-2f1fe62c490b?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=M3w0ODMyOTJ8MHwxfGFsbHx8fHx8fHx8fDE2OTA5NDcxODl8&ixlib=rb-4.0.3&q=80&w=200"
+    expect(page.find("#logo")["src"]).to eq(image_src)
+    expect(page).to have_content("Likes:")
+
```

Comment:
Be sure to copy this test into the item show spec and adjust wording appropriately